examples/traefik/README.md

```diff
@@ -0,0 +1,14 @@
+# Basic configuration to use with the traefik reverse proxy
+
+Note: Tested with traefik 1.7
+
+- Remove the port-binds for the web service
+- The provided example assumes an external network with the name "web"
+- I've added comments where to adjust the docker-compose.yml, but I assume that you know your stuff, if you're running traefik.
+
+Uncomment and set DOCKER_HOST_ADDRESS in .env  
+Im pretty sure, that this is mandatory for the docker-setup and should be clearer in the original README. Could be the proxying, didn't investigate further.
```

Comment:
do not guess what is needed and / or correct!

Reply:
You're right on this. Didn't work for me without it, doesn't mean it's mandatory. Will change that.